Objects/call.c

```diff
@@ -942,22 +948,14 @@ PyObject_CallFunction(PyObject *callable, const char *format, ...)
 PyObject *
 PyEval_CallFunction(PyObject *callable, const char *format, ...)
```

Comment:
Main purpose is reducing maintenance cost rather than optimization.
There are very few callers in the world.

I hope compiler can remove duplicated code too, but I'm not sure.